README.md

```diff
@@ -0,0 +1,3 @@
+# Artikolo
+
+An iOS Read it Later app.
```

Comment:
It's a "Read It Soon" app.

👀

![screen shot 2017-04-30 at 6 42 46 pm](https://cloud.githubusercontent.com/assets/526054/25568629/d1475538-2dd4-11e7-9f76-4820596ed59e.png)